forward_engineering/utils/assignTemplates.js

```diff
@@ -0,0 +1,17 @@
+const template = (modifiers = '') => new RegExp('\\$\\{(.*?)\\}', modifiers);
+const getAllTemplates = str => str.match(template('gi')) || [];
+const parseTemplate = str => (str.match(template('i')) || [])[1];
+
+const assignTemplates = (str, templates) => {
```

Comment:
any chance to transform the params into a property bag? (and describe JSdoc as well)